### PR TITLE
Fixes to global page

### PR DESCRIPTION
**Pull request by stanleypliu**
General fixes for the home page - bug fixes and a rework of the 'Habitat Cover' section.

### CODE REVIEW

File: app/serializers/serializers/habitat_cover_serializer.rb [hunk 1]
```diff
@@ -10,20 +10,44 @@ def serialize
...
+      habitat_global_change = Habitat.find_by(name: habitat.to_s).global_change_stat
+
+      if habitat_global_change.nil?
+        change_percentage = '-'
+      else
+        citations = habitat_global_change.global_change_citations.to_a.first
+
+        if habitat_global_change[:percentage_lost].nil?
+          change_percentage = habitat_global_change[:lower_bound_percentage].round().to_s + ' - ' +
+          habitat_global_change[:upper_bound_percentage].round().to_s + '%'
```
[the remark this answers is not in the document]
Comment (stanleypliu) [reply app/serializers/serializers/habitat_cover_serializer.rb]: Yes, it does end up in a very long line though.

File: app/serializers/serializers/habitat_cover_serializer.rb [hunk 2]
```diff
@@ -10,48 +10,59 @@ def serialize
...
+        citation: stats[:citations].nil? ? stats[:citation] : stats[:citations][:citation]
         )
-    else
-      citation = I18n.t('home.habitat_cover.no_citation')
-    end
 
-      habitat_cover_item[:change_percentage] = change_percentage
+      if stats[:citations]
+        if stats[:citations][:citation_url].nil?
+          citation_url = I18n.t("home.habitat_cover.no_url")
```
Comment (lucacug) [inline app/serializers/serializers/habitat_cover_serializer.rb:31]: I'm still getting a broken link when no url is specified, I think you need to do something similar to the Known habitat occurrence component in the country page, changing the yml file

Comment (stanleypliu) [reply app/serializers/serializers/habitat_cover_serializer.rb:31]: Sorry Luca, I forgot to say that I destroyed all instances of `GlobalChangeStat`, then ran the rake task to rebuild my records! Then you should have the modal like this: 
Screenshot 2020-06-18 at 11 24 43

**Comment by lucacug** [comment]
Happy now, merging. Just to remind that after the deploy you will need to run the import for the global change stats.

**Pull request merged**